app/[locale]/gallery/collections/(noirlab)/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, PropsWithChildren } from "react";
+import Container from "@rubin-epo/epo-react-lib/Container";
+import Stack from "@rubin-epo/epo-react-lib/Stack";
+import MediaPolicy from "@/components/organisms/gallery/MediaPolicy";
+
+const NOIRLabAssetLayout: FC<PropsWithChildren<LocaleProps>> = async ({
```

Comment:
use a [route group](https://nextjs.org/docs/app/building-your-application/routing/route-groups) for NOIRLab assets